.github/workflows/cd.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   build:
+    if: contains(toJson(github.event.commits), '[skip ci]') == false
```

Comment:
🤔 I wonder if it should be `[skip cd]` here, since technically this action rule is for deployment, not integration... maybe the label should say `[skip cicd]`, or have 3 variations, `[skip ci]`, `[skip cd]`, and `[skip cicd]`?

Reply:
I was thinking that if it's a doc change, it doesn't need to be tested or deployed. If it's not a doc change, it needs to be tested and deployed. Hence a single [skip ci]. but maybe I'm missing something.

At any rate, I don't think I can test the change in the cd file until it's merged into master, due to the branch rule at the top.

Reply:
That makes sense 👍🏻